pallets/simple-nft/src/weights.rs

```diff
@@ -28,16 +28,16 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
     fn run_process(i: usize, o: usize) -> Weight {
-		(14_049_000 as Weight)
-			// Standard Error: 7_000
-			.saturating_add((12_902_000 as Weight).saturating_mul(i as Weight))
-			// Standard Error: 7_000
-			.saturating_add((5_897_000 as Weight).saturating_mul(o as Weight))
-			.saturating_add(T::DbWeight::get().reads(1 as Weight))
-			.saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(i as Weight)))
-			.saturating_add(T::DbWeight::get().writes(1 as Weight))
-			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(i as Weight)))
-			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(o as Weight)))
+        (14_049_000 as Weight)
+            // Standard Error: 7_000
+            .saturating_add((12_902_000 as Weight).saturating_mul(i as Weight))
+            // Standard Error: 7_000
+            .saturating_add((5_897_000 as Weight).saturating_mul(o as Weight))
+            .saturating_add(T::DbWeight::get().reads(1 as Weight))
+            .saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(i as Weight)))
+            .saturating_add(T::DbWeight::get().writes(1 as Weight))
+            .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(i as Weight)))
+            .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(o as Weight)))
```

Comment:
Hang on did the weights really not change at all?

Reply:
this is some churn as I forgot to `cargo fmt` after changing weights on the previous PR. I think in a distant call in 2021 we decided there was no need to update benchmarking for this PR

Reply:
The benchmarking doesn't need to change no, but the weights will likely be different so you just need to run the benchmarking suite again and re-integrate

Reply:
ran again and got slightly different weights

Reply:
Should we automate the benchmarking for outputting the weights?

Reply:
It would be nice but it's complicated because the benchmarking output is not a 1-2-1 with what ends up going here. I think we should have a backlog item for this

Reply:
https://digicatapult.atlassian.net/browse/IN-66